### ISSUE TITLE
KC Chatcommand Not Working On All Rooftops

### ISSUE THREAD

**Issue by fr0gdude** [runelite/runelite#16321]
### Type

Incorrect behavior

### Operating System

Windows

### OS Version

Windows 10

### Bug description

Some rooftops do not show lap count with !kc chat command.

I have noticed the issue with Draynor Village and Al Kharid. I have not tested all of them due to a lack of agility level. 

### Screenshots or videos

![image](https://user-images.githubusercontent.com/105972849/221248330-7903e643-03aa-47eb-b859-9e0ed964fd0b.png)

### RuneLite version

RuneLite Version 1.9.11.2
Oldschool revision: Rev 211
Launcher version: 2.5.0

### Logs

[client.log](https://github.com/runelite/runelite/files/10827279/client.log)
[launcher.log](https://github.com/runelite/runelite/files/10827280/launcher.log)

**Comment by Nightfirecat** [comment]
This is likely because you have not completed these agility courses recently, I'm able to get all of these commands to work fine. If you complete a lap of each of these courses, you should find these commands will work afterwards.

**Comment by fr0gdude** [comment]
@Nightfirecat I completed them only a few days ago. Is that expected behavior?

**Comment by Nightfirecat** [comment]
No, those should work. What happens if you complete another lap of each of those courses and use the associated `!kc` command immediately afterward?
